package.json

```diff
@@ -39,7 +39,7 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "eslint": "^7.32.0",
+    "eslint": "^8.3.0",
```

Comment:
Let's update the eslint to ^8.11.0 through the PRs.

Reply:
OK